### PR TITLE
Cross-mio updates - 2024 platforms

### PR DESCRIPTION
**Pull request by rtravis-whoi**
Added 2024 platforms for Pioneer MAB, Irminger, Station Papa
Inserted NES into 'old' Pioneer platforms and made a few description corrections for consistency

My working spreadsheet is attached so you can see main updates in red font, plus I tweaked some old Pioneer descriptions for consistency.

[platform_bulk_load-AssetRecord_2024-01-29.xlsx](https://github.com/WHOIGit/ooicgsn-asset-management/files/14087648/platform_bulk_load-AssetRecord_2024-01-29.xlsx)

### CODE REVIEW

File: bulk/platform_bulk_load-AssetRecord.csv [hunk 1]
```diff
@@ -220,156 +220,176 @@ CGMCE-09OSSM-00014,,Platform,0,Coastal Endurance Washington Offshore Surface Moo
...
+CGMCP-04OSSM-00010,,Platform,0,Coastal Pioneer NES Offshore Surface Mooring,MOORING PIONEER NES OS SURFACE,WHOI,CP04OSSM,CP04OSSM-00010,,,,,Woods Hole Oceanographic Institution
+CGMCP-04OSSM-00011,,Platform,0,Coastal Pioneer NES Offshore Surface Mooring,MOORING PIONEER NES OS SURFACE,WHOI,CP04OSSM,CP04OSSM-00011,,,,,Woods Hole Oceanographic Institution
+CGMCP-04OSSM-00012,,Platform,0,Coastal Pioneer NES Offshore Surface Mooring,MOORING PIONEER NES OS SURFACE,WHOI,CP04OSSM,CP04OSSM-00012,,,,,Woods Hole Oceanographic Institution
+CGMCP-04OSSM-00013,,Platform,0,Coastal Pioneer NES Offshore Surface Mooring,MOORING PIONEER NES OS SURFACE,WHOI,CP04OSSM,CP04OSSM-00013,,,,,Woods Hole Oceanographic Institution
+CGMCP-04OSSM-00014,,Platform,0,Coastal Pioneer NES Offshore Surface Mooring,MOORING PIONEER NES OS SURFACE,WHOI,CP04OSSM,CP04OSSM-00014,,,,,Woods Hole Oceanographic Institution
+CGMCP-04OSSM-00015,,Platform,0,Coastal Pioneer NES Offshore Surface Mooring,MOORING PIONEER NES OS SURFACE,WHOI,CP04OSSM,CP04OSSM-00015,,,,,Woods Hole Oceanographic Institution
```
Comment (snwhite) [inline bulk/platform_bulk_load-AssetRecord.csv:372]: Addition of NES to initial Pioneer Array  looks good.

File: bulk/platform_bulk_load-AssetRecord.csv [hunk 2]
```diff
@@ -220,156 +220,176 @@ CGMCE-09OSSM-00014,,Platform,0,Coastal Endurance Washington Offshore Surface Moo
...
+CGMCP-13SOPM-00001,,Platform,0,Coastal Pioneer MAB Southern Profiler Mooring,MOORING PIONEER MAB SO PROFILER,WHOI,CP13SOPM,CP13SOPM-00001,,,,,Woods Hole Oceanographic Institution
+CGMCP-13SOPM-00002,,Platform,0,Coastal Pioneer MAB Southern Profiler Mooring,MOORING PIONEER MAB SO PROFILER,WHOI,CP13SOPM,CP13SOPM-00002,,,,,Woods Hole Oceanographic Institution
+CGMCP-14NEPM-00001,,Platform,0,Coastal Pioneer MAB Northeastern Profiler Mooring,MOORING PIONEER MAB NE PROFILER,WHOI,CP14NEPM,CP14NEPM-00001,,,,,Woods Hole Oceanographic Institution
+CGMCP-14NEPM-00002,,Platform,0,Coastal Pioneer MAB Northeastern Profiler Mooring,MOORING PIONEER MAB NE PROFILER,WHOI,CP14NEPM,CP14NEPM-00002,,,,,Woods Hole Oceanographic Institution
+CGMCP-14SEPM-00001,,Platform,0,Coastal Pioneer MAB Southeastern Profiler Mooring,MOORING PIONEER MAB SE PROFILER,WHOI,CP14SEPM,CP14SEPM-00001,,,,,Woods Hole Oceanographic Institution
+CGMCP-14SEPM-00002,,Platform,0,Coastal Pioneer MAB Southeastern Profiler Mooring,MOORING PIONEER MAB SE PROFILER,WHOI,CP14SEPM,CP14SEPM-00002,,,,,Woods Hole Oceanographic Institution
```
Comment (snwhite) [inline bulk/platform_bulk_load-AssetRecord.csv:392]: New Pioneer MAB moorings look good.

File: bulk/platform_bulk_load-AssetRecord.csv [hunk 3]
```diff
@@ -422,6 +445,7 @@ CGMGI-03FLMB-00007,,Platform,0,Global Irminger Sea Flanking Subsurface Mooring B
 CGMGI-03FLMB-00008,,Platform,0,Global Irminger Sea Flanking Subsurface Mooring B,MOORING MESCALE FLANKING B SP,WHOI,GI03FLMB,GI03FLMB-00008,,,,,Woods Hole Oceanographic Institution
 CGMGI-03FLMB-00009,,Platform,0,Global Irminger Sea Flanking Subsurface Mooring B,MOORING MESCALE FLANKING B SP,WHOI,GI03FLMB,GI03FLMB-00009,,,,,Woods Hole Oceanographic Institution
 CGMGI-03FLMB-00010,,Platform,0,Global Irminger Sea Flanking Subsurface Mooring B,MOORING MESCALE FLANKING B SP,WHOI,GI03FLMB,GI03FLMB-00010,,,,,Woods Hole Oceanographic Institution
+CGMGI-03FLMB-00011,,Platform,0,Global Irminger Sea Flanking Subsurface Mooring B,MOORING MESCALE FLANKING B SP,WHOI,GI03FLMB,GI03FLMB-00011,,,,,Woods Hole Oceanographic Institution
```
Comment (snwhite) [inline bulk/platform_bulk_load-AssetRecord.csv:448]: New Irminger Moorings look good.

File: bulk/platform_bulk_load-AssetRecord.csv [hunk 4]
```diff
@@ -452,6 +478,7 @@ CGMGP-03FLMB-00007,,Platform,0,Global Station Papa Flanking Subsurface Mooring B
 CGMGP-03FLMB-00008,,Platform,0,Global Station Papa Flanking Subsurface Mooring B,MOORING MESCALE FLANKING B SP,WHOI,GP03FLMB,GP03FLMB-00008,,,,,Woods Hole Oceanographic Institution
 CGMGP-03FLMB-00009,,Platform,0,Global Station Papa Flanking Subsurface Mooring B,MOORING MESCALE FLANKING B SP,WHOI,GP03FLMB,GP03FLMB-00009,,,,,Woods Hole Oceanographic Institution
 CGMGP-03FLMB-00010,,Platform,0,Global Station Papa Flanking Subsurface Mooring B,MOORING MESCALE FLANKING B SP,WHOI,GP03FLMB,GP03FLMB-00010,,,,,Woods Hole Oceanographic Institution
+CGMGP-03FLMB-00011,,Platform,0,Global Station Papa Flanking Subsurface Mooring B,MOORING MESCALE FLANKING B SP,WHOI,GP03FLMB,GP03FLMB-00011,,,,,Woods Hole Oceanographic Institution
```
Comment (snwhite) [inline bulk/platform_bulk_load-AssetRecord.csv:481]: New Papa Moorings look good.

**Review by snwhite — APPROVED** [review]

File: bulk/platform_bulk_load-AssetRecord.csv (hunk 1 above)
Comment (cooleyky) [inline bulk/platform_bulk_load-AssetRecord.csv:372]: NES and updated descriptions look good

File: bulk/platform_bulk_load-AssetRecord.csv (hunk 3 above)
Comment (cooleyky) [reply bulk/platform_bulk_load-AssetRecord.csv:448]: Irminger looks good

File: bulk/platform_bulk_load-AssetRecord.csv (hunk 4 above)
Comment (cooleyky) [reply bulk/platform_bulk_load-AssetRecord.csv:481]: Papa looks good

File: bulk/platform_bulk_load-AssetRecord.csv (hunk 2 above)
Comment (cooleyky) [reply bulk/platform_bulk_load-AssetRecord.csv:392]: Pioneer MAB looks good!

**Review by cooleyky — APPROVED** [review]
All new moorings and updates look good!

**Comment by rtravis-whoi** [comment]
[Pioneer 20 AM Updates](https://trello.com/c/22LKGqi6/232-pioneer-20-am-updates)

**Pull request merged**